mssql-plugin/docs/SqlServer-batchsink.md

```diff
@@ -75,6 +75,56 @@ https://docs.microsoft.com/en-us/sql/relational-databases/system-compatibility-v
+    | CHAR             | string                 |                                                                |
+    | DATE             | date                   |                                                                |
+    | DATETIME         | timestamp              |                                                                |
+    | DATETIME2        | timestamp              |                                                                |
+    | DATETIMEOFFSET   | string                 |                                                                |
+    | DECIMAL          | decimal                |                                                                |
+    | FLOAT            | double                 |                                                                |
+    | IMAGE            | bytes                  |                                                                |
+    | INT              | int                    |                                                                |
+    | MONEY            | decimal                |                                                                |
+    | NCHAR            | string                 |                                                                |
+    | NTEXT            | string                 |                                                                |
+    | NUMERIC          | decimal                |                                                                |
+    | NVARCHAR         | string                 |                                                                |
+    | NVARCHAR(MAX)    | string                 |                                                                |
+    | REAL             | float                  |                                                                |
+    | SMALLDATETIME    | timestamp              |                                                                |
+    | SMALLINT         | int                    |                                                                |
+    | SMALLMONEY       | decimal                |                                                                |
+    | TEXT             | string                 |                                                                |
+    | TIME             | time                   | TIME data type has the accuracy of 100 nanoseconds which is    |
+    |                  |                        | not currently supported. Values of this type will be rounded   |
+    |                  |                        | to microsecond.                                                |
+    | TINYINT          | int                    |                                                                |
+    | UDT              | bytes                  | UDT mapped as basic type if it's an alias of this type. 'SSN'  |
```

Comment:
I'm not familiar with the UDT type, but from the comment it seems like some user defined type that is just an alias of something else? So it gets mapped to whatever it is an alias of?

UDT types are mapped according to the type they are an alias of. For example, is there is an 'SSN' type that was created as 'CREATE TYPE SSN FROM varchar(11);', that type would get mapped to a CDAP string. Common Language Runtime UDTs are mapped to CDAP bytes.

Reply:
Yes, you are right. It gets mapped to whatever it is an alias of. 
Type created as `CREATE TYPE BIG_UDT FROM BIGINT NOT NULL` is mapped to `long`. 
Added `BIG_UDT` user defied type to tests, comment changed to:
```
| UDT              | bytes                  | UDT types are mapped according to the type they are an alias   |
|                  |                        | of. For example, is there is an 'SSN' type that was created as |
|                  |                        | 'CREATE TYPE SSN FROM varchar(11);', that type would get       |
|                  |                        | mapped to a CDAP string. Common Language Runtime UDTs are      |
|                  |                        | mapped to CDAP bytes.                                          |
```